syntax = "proto3";

package simplegrpc;

service SimpleGrpc {
  rpc SayHello (HelloReq) returns (HelloRes) {}
  rpc SayHelloToMany (stream HelloReq) returns (HelloRes) {}
  rpc CheckInbox (HelloReq) returns (stream HelloRes) {}
}

message HelloReq {
  // The name of the member(s)
  string name = 1;
}

message HelloRes {
  // The message we want to say to member(s)
  string message = 1;
}